infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/msghandlers/LogReplicationServer.java

```diff
@@ -342,6 +358,15 @@ private void handleLeadershipResponse(RequestMsg req, ResponseMsg response,
                 response.getPayload().getLrLeadershipResponse());
     }
 
+    @LogReplicationResponseHandler(responseType = LR_SINK_SESSION_INITIALIZATION_ACK)
+    private void handleSinkSessionInitialization(RequestMsg req, ResponseMsg response,
```

Comment:
similar to the above - can we rename this method to 'handleSinkSessionCreationResponse'